samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -230,6 +230,16 @@ static constexpr char inference_only_message[] =
     " To enable full mode for static models pass \"false\" value to this argument:"
     " ex. \"-inference_only=false\".\n";
 
+// @brief message for load property option
+static const char load_property_message[] =
+    "Optional. Path to JSON file to load custom property for OpenVINO runtime API 2.0."
+    " Please note, command line parameters have higher priority than parameters from configuration "
+    "file.";
+
+// @brief message for dump property option
+static const char dump_property_message[] =
+    "Optional. Path to JSON file to dump property for OpenVINO runtime API 2.0, which are set by application.";
```

Comment:
- Either 'properties are set' or 'property is set'
- 'by the application.'

Reply:
Ok, I will change it to "property is set".